docs/sphinx/how_to/magma_install/magma_install.md

```diff
@@ -9,4 +9,17 @@ each framework. Refer to the ROCm Compatible Frameworks Release Notes for each
 framework's most current release notes at
 /bundle/ROCm-Compatible-Frameworks-Release-Notes/page/Framework_Release_Notes.html.
 ## PyTorch
-PyTorch is an open source Machine Learning Python library, primarily differentiated by Tensor computing with GPU acceleration and a type-based automatic differentiation. Other advanced features include:
\ No newline at end of file
+PyTorch is an open source Machine Learning Python library, primarily differentiated by Tensor computing with GPU acceleration and a type-based automatic differentiation. Other advanced features include:
+• Support for distributed training
+
+•  Native ONNX support

```

Comment:
@Ehud-Sharlin you have some unicode bullets. You should replace with -

Reply:
Thanks Saad, I know... Just trying out things...
I am experincing long (5 minish) delays between my commits and the time I can see them rendered on https://advanced-micro-devices-rocm--1883.com.readthedocs.build/en/1883/how_to/magma_install/magma_install.html , not sure why...
Anyways, I'll keep working on this! Have a good night

Reply:
Build delays of 5 minutes are expected as the ReadTheDocs pipeline is fairly slow. Locally building docs is another option we should explore.